recipe/0102-Fixed-failure-on-baremetal.patch

```diff
@@ -0,0 +1,49 @@
+From fead0a034c286a5c5f9c56713907e962fca54963 Mon Sep 17 00:00:00 2001
+From: Nishidha Panpaliya <npanpa23@in.ibm.com>
+Date: Mon, 2 Nov 2020 04:06:52 -0600
+Subject: [PATCH] Fixed failure on baremetal
+
+---
+ setup.py | 8 ++++++--
+ 1 file changed, 6 insertions(+), 2 deletions(-)
+
+diff --git a/setup.py b/setup.py
+index 85a6921..4d10f5b 100644
+--- a/setup.py
++++ b/setup.py
+@@ -122,10 +122,13 @@ def get_extensions():
+     define_macros = []
+ 
+     extra_compile_args = {}
++    cuda_include_dir = None
+     if (torch.cuda.is_available() and ((CUDA_HOME is not None) or is_rocm_pytorch)) \
+             or os.getenv('FORCE_CUDA', '0') == '1':
+         extension = CUDAExtension
+         sources += source_cuda
++        cuda_include_dir = os.getenv("CUDA_INCLUDE_PATH", None)
++        print("CUDA Include Path: ", cuda_include_dir)
+         if not is_rocm_pytorch:
+             define_macros += [('WITH_CUDA', None)]
+             nvcc_flags = os.getenv('NVCC_FLAGS', '')
+@@ -150,7 +153,8 @@ def get_extensions():
+     sources = [os.path.join(extensions_dir, s) for s in sources]
+ 
+     include_dirs = [extensions_dir]
+-
++    if cuda_include_dir:
++        include_dirs.append(cuda_include_dir)
+     ext_modules = [
+         extension(
+             'torchvision._C',
+@@ -199,7 +203,7 @@ def get_extensions():
+                     video_reader_src_dir,
+                     ffmpeg_include_dir,
+                     extensions_dir,
+-                ],
++                ] + [cuda_include_dir] if not None else [],
```

Comment:
can we use `cuda_include_dir=[]` in the beginning to avoid this condition .

Reply:
As discussed, we can't make this change as in case of cpu variant, it would break.